styles/components/Error.module.css

```diff
@@ -0,0 +1,6 @@
+.error {
+  display: flex;
+  justify-content: center;
+  font-size: 1.5rem;
+  color: #f37121;
+}
```

Comment:
lets move the styles for Empty, Error, and Loading into `styles/components/util`